src/Analysis/Codelyzer.Analysis.CSharp/CSharpRoslynProcessor.cs

```diff
@@ -175,6 +177,15 @@ public override UstNode VisitExpressionStatement(ExpressionStatementSyntax node)
             return base.VisitExpressionStatement(node);
         }
 
+        private UstNode VisitExpressionSyntax(ExpressionSyntax node)
+        {
+            if (node == null) return null;
+
+            return _expressionSyntaxToVisitMethodMap.TryGetValue(node.GetType().Name, out var visitMethod)
+                ? visitMethod.Invoke(node)
+                : null;
```

Comment:
As discussed, let's try using the HandleGenericVisit here instead of maintaining a list